### PR TITLE
people documentation

### PR DESCRIPTION
**Pull request by pmajhwar**

### CODE REVIEW

File: docs/people/setup.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+id: setup
+title: Setup Developer Account
+sidebar_label: Setup Developer Account
+---
+
+In order to create Apps on Deskera platform, developer needs to register with us as a Partner. 
+
+We uses OAuth 2.0 for the Authentication scheme. Developers must first register with us by providing some details about their app, such as email, callbankURL, and name.
+
+Please send us an email at `care@deskera.com` with the information mentioned below to register as a partner and obtain Deskera `client_id` and `client_secret`
```
Comment (ashutosh091) [inline docs/people/setup.md]: Got this form now to get teh details https://docs.google.com/forms/d/e/1FAIpQLScZpPZg0Gu18AFF4E7YrekqSWLLbN7MVi6fBxTluQ_KFm4-zw/viewform?usp=sf_link

File: docs/people/started.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+id: started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+All [Deskera](https://www.deskera.com/books/) endpoints are secured and can only be reached after a successful authentication. Deskera APIs require a valid token to communicate. This token should be passed as `x-access-token` in the request header while interacting Deskera APIs. To generate a valid token, the user can trade-off username and password with Authentication API. Authentication API will return accessToken in the response body which can be used to communicate with Deskera APIs.
```
Comment (ashutosh091) [inline docs/people/started.md]: i hope you see it is pointing to books url

File: docs/people/setup.md (hunk 1 above)
Comment (pmajhwar) [reply docs/people/setup.md]: made the necessary changes

File: docs/people/started.md (hunk 2 above)
Comment (pmajhwar) [reply docs/people/started.md]: done, please check

**Pull request merged**